minecraft_servers/asvadianetwork/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "server_name": "asvadianetwork",
+  "nice_name": "Asvadia Network",
+  "direct_ip": "mc.asvadia.eu",
+  "server_wildcards": [
+    "%.asvadia.eu"
+  ],
+  "social": {
+    "web": "https://asvadia.eu",
+    "web_shop": "https://boutique.asvadia.eu",
```

Comment:
Not released, please remove

<img width="611" alt="Bildschirmfoto 2023-05-10 um 21 02 23" src="https://github.com/LabyMod/server-media/assets/45363287/1d294b28-409b-4062-865b-9f122b3d2b6e">